<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>사용자 이름</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_finish">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>완료</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="lw_online_user">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>161</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>온라인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅창</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tw_chat_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>441</width>
       <height>241</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>채팅방 이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>방장</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>접속인원</string>
      </property>
     </column>
    </widget>
    <widget class="QStackedWidget" name="sw_open_chat">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_4"/>
     <widget class="QWidget" name="page_5">
      <widget class="QLineEdit" name="le_chat_room">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>17</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>방 제목</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_room_fns">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>42</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>완료</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_chat_open">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방 개설</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="btn_send_message">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>540</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_receiver">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>데이터 AI SW 개발자 양성</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lw_message">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>71</y>
       <width>420</width>
       <height>461</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_senders_name">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>senders name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_message">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>540</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_game">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>540</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Game</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_name</tabstop>
  <tabstop>btn_finish</tabstop>
  <tabstop>btn_send_message</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
